tests/common/file_utils.rs

```diff
@@ -58,3 +63,10 @@ pub fn create_file_with_content(path: &PathBuf, content: &str) -> () {
     file.write_all(content.as_bytes())
         .expect(&format!("cannot write to file {:?}", path));
 }
+
+pub fn read_file(path: &PathBuf) -> String {
+    let mut file = File::create(&path).unwrap();
+    let mut buffer = String::new();
+    std::fs::read_to_string(&mut buffer);
+    buffer
+}
```

Comment:
It seems to me, this function `read_file` is not needed and `std::fs::read_to_string` is sufficient.

Reply:
Yes this is mistake, removing method